src/libLookup/Synchronizer.cpp

```diff
@@ -143,6 +143,16 @@ bool Synchronizer::FetchDSInfo(Lookup* lookup) {
   return true;
 }
 
+bool Synchronizer::FetchInitialDSInfo(Lookup* lookup) {
+  if (LOOKUP_NODE_MODE) {
+    return true;
```

Comment:
Can add a warning message here, so we can find call wrong function bug.

Reply:
ok